### ISSUE TITLE
Error reading data: (12002) Operation time out

### ISSUE THREAD

**Issue by ArabicLawnmower** [intel/openvino-plugins-ai-audacity#294]
![Setup 13_09_2024 14_07_03](https://github.com/user-attachments/assets/b381b2ed-1e6c-4ace-a2a3-d2f04d64d052)

So, after starting the download for OpenVino, everything goes right... it's a bit slow, i went to sleep to wait through the night. i know it did load, in fact it installed (but failed as shown here) many parts, as the green bar went to the end and restarted. so, it did download things, but it seems like at a point it fails...

i got the link where all the zips are shown, is there a way i can download EVERYTHING and set them in audacity?

**Comment by RyanMetcalfeInt8** [comment]
Yes, if you are having trouble getting the installer to download the models correctly, you can download them & populate the ```C:\Program Files\openvino-models``` folder manually.

You can look here (https://github.com/intel/openvino-plugins-ai-audacity/tree/main/doc/build_doc/windows#installing-the-openvino-models) for the commands that can manually assemble this openvino-models folder with the right content & structure.

Let me know if you need help.

Thanks,
Ryan

**Comment by ArabicLawnmower** [comment]
Hello, thanks for answering. Yes, if possible i'd like some help with manually setting it up. do i have to estract every zip inside the folder?

by the way, i moved Audacity inside D:\(custom folder), and it's not in C:\Program Files anymore (because i was running out of space). do i have to move it back in to work? and also, i don't have the open-vino folder...

**Comment by ArabicLawnmower** [comment]
Hello? can i get some support?

**Comment by RyanMetcalfeInt8** [comment]
Hi @ArabicLawnmower,

Yes, you will need to create an ```openvino-models``` folder, and then populate it with the files inside of the zip. You don't necessarily need to populate the folder will *all* models -- only for the features that you are interested in (like Music Separation, Whisper Transcription, etc.).

> by the way, i moved Audacity inside D:(custom folder), and it's not in C:\Program Files anymore (because i was running out of space). do i have to move it back in to work? 

It shouldn't really matter where your Audacity installation is located. ```D:\SomeCustomFolder``` is fine. 

> and also, i don't have the open-vino folder...
After running our installer and choosing 'No Models' from the drop-down list, there should be an ```openvino-models``` folder sitting in the same directory as Audacity.exe. This will be empty of course, and you can delete it / replace it with the ```openvino-models``` folder that you will be creating / populating from the instructions in the link that I gave above.

**Comment by ArabicLawnmower** [comment]
Thank you for telling me, i did all the steps. i added every model, but inside audacity i don't see any of them. where are they located?

**Comment by RyanMetcalfeInt8** [comment]
Here is what my ```openvino-models``` folder looks like, for example:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/98cb27d2-5d43-4fe5-9942-ef2e5df025be">

Can you double check that yours looks similar?

**Comment by ArabicLawnmower** [comment]
![image](https://github.com/user-attachments/assets/f8df36d0-142a-4c92-af9e-2b4dd738d4d8)
Mine looks like this

**Comment by RyanMetcalfeInt8** [comment]
The content of ```openvino-models``` looks okay -- as long as you are only interested in running *Transcription* and/or *Music Separation*.

> inside audacity i don't see any of them

Do you mean that you don't see the effects as selectable from the Audacity menu? If so, perhaps you need to enable the module in Preferences->Modules. In this case you would set ```mod-openvino``` to enabled, and then restart Audacity.

**Comment by ArabicLawnmower** [comment]
thank you, i seems to work... what about the other things? i'd like to get everything that openvino offers.

**Comment by ArabicLawnmower** [comment]
(sorry if i'm asking twice, it's been a week so i thought it was ok to ask again)

how can i get everything else too?

**Comment by RyanMetcalfeInt8** [comment]
My bad, forgot to respond to this one.

So, the link that I gave to the instructions above (also here: https://github.com/intel/openvino-plugins-ai-audacity/tree/main/doc/build_doc/windows#installing-the-openvino-models) should be what you need to execute (from a cmd.exe window) to populate a ```openvino-models``` folder with the right files. Specifically, take a look at MusicGen and Noise Suppression. Let me know if something isn't clear...

**Comment by chocomilk877** [comment]
hello i have the same error but i not  a programmer and i understand how to follow the steps im very lost
